builtin/providers/aws/resource_aws_emr_task_group.go

```diff
@@ -0,0 +1,114 @@
+package aws
+
+import (
+	"log"
+
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/emr"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsEMRTaskGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsEMRTaskGroupCreate,
+		Read:   resourceAwsEMRTaskGroupRead,
+		Update: resourceAwsEMRTaskGroupUpdate,
+		Delete: resourceAwsEMRTaskGroupDelete,
+		Schema: map[string]*schema.Schema{
+			"cluster_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"instance_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"instance_count": &schema.Schema{
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  60,
+			},
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
what happens if you don't specify a `name` here? From the code below, we still try to send `""`, is the API fine with this?